docs/docs/api/06-Configuration/askui-ui-controller-client.md

```diff
@@ -0,0 +1,74 @@
+---
+custom_edit_url: null
+---
+
+# askui UI Controller Client
+
+## Table of Contents
+
+### Properties
+
+ [askui UI Controller Client](#askui-ui-controller-client)
+  - [Table of Contents](#table-of-contents)
+    - [Properties](#properties)
+  - [Properties](#properties-1)
+    - [uiControlServerUrl](#uiControlServerUrl)
+    - [inferenceServerUrl](#inferenceServerUrl)
+    - [annotationLevel](#annotationLevel)
+    - [credentials](#credentials)
+  - [Example](#example)  
+
+## Properties
+
+### uiControlServerUrl
+
+• **uiControlServerUrl**: `string` - Default: `http://localhost:6769`
+
+This provides the port where the askui UI Controller runs. The Client is also connecting
+to this address.
+
+___
+
+### inferenceServerUrl
+
+• **inferenceServerUrl**: `string` - Default: `https://inference.askui.com`
+
+The address of the askui Inference server.
+___
+
+### annotationLevel
+
+• **annotationLevel**: `AnnotationLevel` - Default: `AnnotationLevel.DISABLED`
+
+AnnotationLevel is implemented as an enum. You have three options: `DISABLED`, `ON_FAILURE`, `ALL`.
+
+`ALL`:  We run the [annotate](../../general/05-Tooling/annotate-image.md) command after the execution of each test step.
+
+
+`ON_FAILURE`: Runs the annotate command if the test step fails 
+
+
+`DISABLED`: Never runs the annotate command after test steps
+
+___
+
+### credentials
+
+• **credentials**: `string` - Default: `envCredentials`
+
+We need to provide this credentials for the http client. Per Default we use the credentials from the user environment.
```

Comment:
Could you document the environment variables and how this can be set using an object? `envCredentials` is an implementation detail and not the default. It should not be documented :laugh